zebra-state/src/service.rs

```diff
@@ -718,98 +635,53 @@ impl StateService {
     /// Attempt to validate and commit all queued blocks whose parents have
     /// recently arrived starting from `new_parent`, in breadth-first ordering.
```

Comment:
Nitpick: Can you update this method documentation?

We might also want to rename the method so it's easy to understand what it does.
What about `send_ready_non_finalized_queued()`?

Reply:
Updated the fn name and documentation in [`a27aa3f`](https://github.com/ZcashFoundation/zebra/pull/5257/commits/a27aa3fa449dd9d9bd482f0de627f349c738d3b7)